layouts/partials/scripts.html

```diff
@@ -60,17 +39,22 @@
 {{ $jsBs := resources.Get "vendor/bootstrap/dist/js/bootstrap.bundle.js" -}}
 {{ $jsBase := resources.Get "js/base.js" -}}
 {{ $jsSearch := resources.Get "js/search.js" | resources.ExecuteAsTemplate "js/search.js" .Site.Home -}}
-{{ $jsMarkmap := resources.Get "js/markmap.js" | resources.ExecuteAsTemplate "js/markmap.js" . -}}
 {{ $jsPlantuml := resources.Get "js/plantuml.js" | resources.ExecuteAsTemplate "js/plantuml.js" . -}}
 {{ $jsDrawio := resources.Get "js/drawio.js" | resources.ExecuteAsTemplate "js/drawio.js" . -}}
 {{ if .Site.Params.offlineSearch -}}
   {{ $jsSearch = resources.Get "js/offline-search.js" -}}
 {{ end -}}
 
-{{ $jsArray := slice $jsBs $jsBase $jsSearch $jsPlantuml $jsMarkmap $jsDrawio -}}
+{{ $jsArray := slice $jsBs $jsBase $jsSearch $jsPlantuml $jsDrawio -}}
 
 {{ if .Page.Store.Get "hasmermaid" -}}
-{{- partial "scripts/mermaid.html" . -}}
+  {{- partial "scripts/mermaid.html" . -}}
+{{ end -}}
+
+{{ if .Page.Store.Get "hasmarkmap" -}}
+  {{ $jsMarkmap := resources.Get "js/markmap.js" | resources.ExecuteAsTemplate "js/markmap.js" . -}}
```

Comment:
After making the suggested changes for `js/markmap.js`, you won't need to run `resources.ExecuteAsTemplate` anymore, so you can drop that.

Reply:
Dropped `resources.ExecuteAsTemplate`, as suggested.